client/influxdb.go

```diff
@@ -124,11 +124,15 @@ func (c *Client) Write(bp BatchPoints) (*Response, error) {
 
 	var b bytes.Buffer
 	for _, p := range bp.Points {
-		if p.Raw != "" {
+		if p.Raw != "" && len(bp.Tags) == 0 {
```

Comment:
Not exactly sure what `Raw` is but I think `Raw` and `BatchPoints.Tags` can't really be used together unless we parse the `Raw` value and add the tags to it.  If you set `Raw` and have `BatchPoints.Tags`, I believe the write will fail because the `Raw` value will not be marshaled... only the `BatchPoints.Tags`.  

Is that intended by this line?